client/src/ui/BitbakeStatusBar.ts

```diff
@@ -74,7 +74,7 @@ export class BitbakeStatusBar {
     if (this.commandInProgress !== undefined) {
       let displayText = 'Building...'
       if (this.commandInProgress.includes('devtool')) displayText = 'Devtool...'
-      if (this.commandInProgress.includes('which devtool')) displayText = 'Scanning...'
+      if (this.commandInProgress.includes('which devtool') || this.commandInProgress.includes('-e')) displayText = 'Scanning...'
```

Comment:
Since this will replace parse on save, I'd rather have this detected as parsing. I recommend to edit lines 47 and 53 to include the name of the -e task for detecting parsing.
```Typescript
    vscode.tasks.onDidStartTask((e: vscode.TaskStartEvent) => {
      if (e.execution.task.name === 'Bitbake: Parse' && e.execution.task.source === 'bitbake') {
      if (e.execution.task.name === 'Bitbake: Parse' && e.execution.task.source === 'bitbake') {
        this.parsingInProgress = true
        this.parsingInProgress = true
        this.updateStatusBar()
        this.updateStatusBar()
      }
      }
    })
    })
```

Reply:
Done.